src/components/Article/SocialButtons.tsx

```diff
@@ -1,10 +1,14 @@
-import { ChatCircleDots, Heart } from "@phosphor-icons/react";
+import { ChatCircleDots, DotsThreeOutline, Heart } from "@phosphor-icons/react";
 
 type SocialButtonsProps = {
   article: Article;
   onCommentsClick?: () => void;
 };
 
+const copyToClipboard = (article: Article) => {
+  navigator.clipboard.writeText(article.url)
```

Comment:
Está cumprindo o propósito, mas acho que seria legal um feedback pro usuário de que o link foi copiado com sucesso, só pra ele não ficar em dúvida se funcionou ou não. Até pode ser algo mais trabalhado se quiser, mas acho quem um alert já é bom

```suggestion
  navigator.clipboard.writeText(article.url)
  alert(t('copiedSuccesfully'))
```

Inclusive teria que mover a função pra dentro do componente pra utilizar a tradução